src/naive/date.rs

```diff
@@ -2237,16 +2237,19 @@ mod rustc_serialize {
     }
 
     #[cfg(test)]
-    use rustc_serialize::json;
+    mod tests {
```

Comment:
Can you fix up the commit message on this one? Seems to be okay on the other one.